api/tests/unit/audit/test_unit_audit_signals.py

```diff
@@ -1,87 +1,76 @@
-import json
-
-from flag_engine.api.document_builders import build_environment_document
+from pytest_django.asserts import assertQuerysetEqual
```

Comment:
I hate that this is camelcase 😂 

Is it bad that I'd kind of like us to do this:

```python
from pytest_django.asserts import assertQuerysetEqual as assert_queryset_equal
```